install-deps.sh

```diff
@@ -146,19 +146,32 @@ function install_pkg_on_ubuntu {
     fi
 }
 
+boost_ver=1.82
+
+function clean_boost_on_ubuntu {
+    local installed_ver=$(apt -qq list --installed ceph-libboost*-dev 2>/dev/null |
+                              grep -e 'libboost[0-9].[0-9]\+-dev' |
+                              cut -d' ' -f2 |
+                              cut -d'.' -f1,2)
+    $SUDO rm -f "/etc/apt/sources.list.d/ceph-libboost${installed_ver}.list"
```

Comment:
Does nothing, it prints stuff then exits successfully.

Reply:
In case how we query `installed_ver` breaks I'm using it *only* to decide whether to bail out early in `clean_boost_on_ubuntu()` and then only if it has no spaces. It still functions as it did in `install_boost_on_ubuntu()` because by that point I actually am confident there is zero or one version installed.